guide/interactions/buttons.md

```diff
@@ -206,7 +206,7 @@ In addition to deferring an interaction response, you can defer the button, whic
 <!-- eslint-skip -->
 
 ```js {7-9}
-const wait = require('util').promisify(setTimeout);
+const wait = require('timers/promises').setTimeout;
```

Comment:
```suggestion
const wait = require('node:timers/promises').setTimeout;
```

Reply:
I first added the `node:` protocol but then I saw that it's not used anywhere in the guide...
Not sure if it's ok if I add that here or if it's better to add all the `node:` protocols missing in another PR

Reply:
I'd say if there are more changes needed for `node:` outside of this pull request, do it separately.